board/main.c

```diff
@@ -627,7 +643,15 @@ int main() {
           }
           break;
         case USB_POWER_CDP:
-#ifndef EON
+#ifdef EON
+          if (power_save_status == POWER_SAVE_STATUS_ENABLED) {
```

Comment:
I think `POWER_SAVE_STATUS_ENABLED` can only be reached if `#ifdef EON` is true, so I'll just remove it.